Workshop Refactor: Machine Learning - Linear Regression (#362)
NuevoFoundation/workshops: content/english/ml-machine-learning/03-confidence-intervals.md

Why is this happening? Why are the **intercept** and **coefficient** values different every time?
{{% expand "**Click to show answer**" %}} 
This is because every time you run the program, a random sample of 30 records is selected. On any random sample you run, both values will be different. But the good thing is, all these values are obtained from the same data set.
{{% /expand %}}

How can we say that we are confident that the values we getting represent the data we have?

# Confidence Interval

The previous example teaches us that, when you work with samples, the **intercept** and **coefficient** values are not necessarily the true ones. We know that the actual value must lie between 2 values but we don't know what they are! This is known as the confidence interval.

When you run the code from **Exercise 1** several times you can see that sometimes the value of the **coefficient** changes every time but notice how it rarely drops below 9000 or is rarely above 10000. What is the average difference between all the possible **coefficients** we can find running the code multiple times? What is that value? This is known as the **standard error**.

> **Standard Error** - explains how different is data is compared to the average of the data.

Reviewer: ```suggestion
**Standard Error** - explains the variation of the data compared to the average of the data.
```
